src/ObjectFactory.php

```diff
@@ -129,6 +129,7 @@ public function convertChangeEvents($changeLogToken, ObjectListInterface $object
     public function convertContentStream(StreamInterface $contentStream)
     {
         // TODO: Implement convertContentStream() method.
```

Comment:
This still needs to be implemented or at least officially short-circuited. I checked the Java implementation and it recreates the stream - we may not need to do that since we natively use the Guzzle stream wrappers.